qrcodegen.c

```diff
@@ -21,8 +21,7 @@
  *   Software.
  */
 
-// We've replaced assert() with an exception generator.
-//#include <assert.h>
+#include <assert.h>
```

Comment:
please keep old code for handling asserts

Reply:
https://github.com/coinkite/mpy-qr/pull/9